dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -213,8 +206,8 @@ def from_no_reply
     email_address('Code.org', 'noreply@code.org')
   end
 
-  def from_hadi
-    email_address('Hadi Partovi', 'hadi_partovi@code.org')
+  def from_survey
```

Comment:
Because [email_address](https://github.com/code-dot-org/code-dot-org/blob/eaa1a3c5734cd836bda1bdc89b3071c94c58e30c/dashboard/app/mailers/pd/workshop_mailer.rb#L195-L199) is a function and does a bit of work.